tests/ballerina-compiler-plugin-test/src/test/resources/META-INF/services/org.ballerinalang.compiler.plugins.CompilerPlugin

```diff
@@ -1,2 +1,2 @@
-io.ballerina.test.compiler.plugins.ABCCompilerPlugin
-io.ballerina.test.compiler.plugins.XYZCompilerPlugin
\ No newline at end of file
+io.ballerina.test.compiler.plugins.TestCompilerPlugin
+io.ballerina.test.compiler.plugins.FunctionsTestCompilerPlugin
```

Comment:
Is this not ending with a new line expected here?

Reply:
It is an auto-generated file.